### ISSUE TITLE
Accessibility (or lack thereof)

### ISSUE THREAD

**Issue by Squeng** [Squoss/Fixadat#4]
https://www.heise.de/hintergrund/Accessibility-im-Web-Teil-1-Erstellen-einer-Webanwendung-7368850.html

**Comment by Squeng** [comment]
https://www.heise.de/hintergrund/Accessibility-im-Web-Teil-2-Barrierefreie-Inhalte-7467924.html
